profile/src/main/java/org/openedx/profile/presentation/settings/SettingsFragment.kt

```diff
@@ -126,7 +126,7 @@ class SettingsFragment : Fragment() {
                             IAPAction.ACTION_RESTORE -> {
                                 IAPDialogFragment.newInstance(
                                     IAPFlow.RESTORE,
-                                    IAPAnalyticsScreen.PROFILE.screenName
+                                    IAPFlowSource.PROFILE.screen
```

Comment:
The `getIAPEventParams()` method returns emptyMap() because we do not have the course ID in the restore flow. Due to this the success event has no properties.

Reply:
we don't need these these params for restore/silent flow.